src/xcode/ENA/ENA.xcodeproj/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>FILEHEADER</key>
+	<string>//
+// 🦠 Corona-Warn-App
+//</string>
+</dict>
+</plist>
```

Comment:
Should this be added here?

Reply:
removed file

Reply:
File added again, as we decided to use the new default header.